deploy/openj9-chart/README.md

```diff
@@ -0,0 +1,106 @@
+|arch            |                                         |CPU architecture preference                                                  |`amd64` or `ppc64le`                                       |
+|image           |repository                               |Image repository                                                             |Defaults to `docker.io/adoptopenjdk`                       |
+|                |tag                                      |Image tag                                                                    |Defaults to `openj9`                                       |
+|                |pullPolicy                               |Image Pull Policy                                                            |`Always`, `Never`, or `IfNotPresent`. Defaults to Always   |
+|container       |command                                  |Container start command / entrypoint, override existing entrypoint if present|Defaults to `jitserver`                                    |
+|                |OPENJ9_JAVA_OPTIONS                      |Options passed into OpenJ9 JVM                                               |                                                           |
+|                |limits.memory                            |Maximum memory allocated for container                                       |Defaults to `8 Gi`                                         |
+|                |limits.cpu                               |Maximum CPU allocated for container                                          |Defaults to `8`                                            |
+|                |requests.memory                          |Minimum memory allocated for container                                       |Defaults to `256 Mi`                                       |
+|                |requests.cpu                             |Minimum CPU allocated for container                                          |Defaults to `1`                                            |
+|service         |port                                     |Port number that java acceleration listens to                                |Defaults to `38400`                                        |
+|replicaCount    |                                         |Number of replica to be deployed                                             |Defaults to `1`                                            |
+
+## Documentation
+
+* See openj9 website: https://www.eclipse.org/openj9/index.html
+* See openj9 jitserver: https://www.eclipse.org/openj9/docs/jitserver/
+* See openj9 repo: https://github.com/eclipse/openj9
+* See openj9 docs: https://github.com/eclipse/openj9-docs
+
+## Limitations
+
+* Deploys on x86 (amd64) and power (ppcle) architecture only, it is intended to support other architectures in future releases.
+* Supports Java 8, Java 11 and Java 14 only, it is intended to support other java versions in future releases.
```

Comment:
```suggestion
* Supports Java 8, Java 11 and the current Java release only, it is intended to support other java versions in future releases.
```

Reply:
Thanks, this has been fixed in latest commit.